finals-2024-01-15/code/4911-luka-skof/src/app/footer/footer.component.html

```diff
@@ -1,5 +1,5 @@
 <p style="border: dashed 1px blue;">
     Footer, put the student info here
     <br />
-    {{student?.id}} - {{student?.name}}
+    {{student?.id}} - {{student?.name}} - {{student?.year}} -
```

Comment:
The student model does not have a `year` property (and it should not have one) - so this will fail.